plugins/discovery-ec2/src/test/java/org/elasticsearch/discovery/ec2/Ec2DiscoveryTests.java

```diff
@@ -50,6 +62,25 @@
 
     protected static ThreadPool threadPool;
     protected MockTransportService transportService;
+    private Map<String, TransportAddress> poorMansDNS = new ConcurrentHashMap<>();
+
+    private static final AtomicInteger portGenerator = new AtomicInteger();
+    private static final InetAddress META_ADDRESS;
```

Comment:
It's a copy and paste from here: https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/common/transport/TransportAddress.java#L43

I'd prefer keeping the same name as we have in master branch.